web-components/src/components/modal/wallet-modal/components/WalletModal.Mobile.config.tsx

```diff
@@ -0,0 +1,43 @@
+import QRCode from 'qrcode.react';
+
+import { Center } from '../../../../components/box';
+import ContainedButton from '../../../../components/buttons/ContainedButton';
+import { IconTabProps } from '../../../../components/tabs/IconTab';
+import { Subtitle } from '../../../../components/typography';
+
+type getWalletModalMobileTabsParams = {
+  mobileConnectUrl?: string;
+  uri?: string;
+};
+
+export const getWalletModalMobileTabs = ({
+  mobileConnectUrl,
+  uri,
+}: getWalletModalMobileTabsParams): IconTabProps[] =>
+  [
+    mobileConnectUrl
+      ? {
```

Comment:
could we switch this to something other than a ternary operator? i find ternary operators that involve a bigger chunk of code like this hard to read.

Reply:
Completely agree with that. Had to use it here because it's an array definition and not a function.
I think I'll move this logic to the filter part to make it more clear.

Reply:
thanks for changing this, i think it is more readable with the filter